README.md

```diff
@@ -9,7 +9,7 @@ A deployment of SeaClouds can be launched using Apache Brooklyn. We currently su
 
 ## Launching with Vagrant
 
-Make sure you have [Vagrant](https://www.vagrantup.com/) and [Apache Brooklyn](https://brooklyn.incubator.apache.org/download/index.html) installed, then:
+Make sure you have [Vagrant](https://www.vagrantup.com/), [Virtual Box](https://www.virtualbox.org/) and [Apache Brooklyn](https://brooklyn.incubator.apache.org/download/index.html) installed, then:
```

Comment:
virtualbox is only one of the hypervisor supported by vagrant, I think it is the default when you install vagrant, but if you think it is useful in the doc, I'm fine